Classes/Security/CheckPermissions.php

```diff
@@ -301,7 +277,7 @@ public function matchFeGroupsWithFeUser($groups, $userFeGroups)
         }
 
         // enabled for all loggedIn Users
-        if (str_contains($groups, '-2')) {
+        if (strpos($groups, '-2') !== false) {
```

Comment:
This is not needed, as TYPO3 11 uses symfony polyfill for PHP 8.0 functions. So you can use str_contains, even if using PHP 7.4. This is also used in the core (see https://forge.typo3.org/issues/95466)